aws/logs_monitoring/template.yaml

```diff
@@ -154,6 +154,10 @@ Parameters:
       - true
       - false
     Description: Set to true to deploy the Forwarder to a VPC and send logs, metrics, and traces via a proxy. When set to true, must also set VPCSecurityGroupIds and VPCSubnetIds.
+  DdHttpProxyURL:
```

Comment:
Can you add this parameter to the README? Both the parameter reference table and the section discussing proxy options?